.tools/readmes/README.md

```diff
@@ -107,7 +109,7 @@ Any content you add within these comments is preserved in subsequent generations
 of the README. Do not change the names of these comments or remove them. Keep them
 empty if you don't need custom content.
 
-## Generate multiple READMEs (EXPERIMENTAL)
+## Generate multiple READMEs
 
 [`multi.py`](multi.py) executes the WRITEME logic across a set of languages,
 versions, and services.
```

Comment:
Can we also rename multi.py to writeme.py? Multi only makes sense when writeme exists.

Reply:
An excellent suggestion. Done!